### PR TITLE
Add stylelint setup instructions for PHP Storm

### PR DESCRIPTION
**Pull request by hanakin**

### CODE REVIEW

**Comment by VSEphpbb** [comment]
FYI: I have had issues with this on PHPStorm.

When it's installed locally (i.e.: node-modules is inside phpBB) it works fine when setting up PHPStorm's Stylelint settings to find it in the project.

However, when stylelint AND stylelint-order are installed globally, and trying to set PHPStorm to use the globally installed stylelint package (and no node_modules folder is in the phpbb project), it fails to work in the IDE with the error:

```
/usr/local/bin/node /usr/local/lib/node_modules/stylelint/bin/stylelint.js -f json
Error: Could not find "stylelint-order". Do you need a `configBasedir`?
    at module.exports (/usr/local/lib/node_modules/stylelint/lib/utils/configurationError.js:8:27)
    at module.exports (/usr/local/lib/node_modules/stylelint/lib/utils/getModulePath.js:15:11)
    at /usr/local/lib/node_modules/stylelint/lib/augmentConfig.js:139:14
    at Array.map (native)
    at absolutizePaths (/usr/local/lib/node_modules/stylelint/lib/augmentConfig.js:138:48)
    at /usr/local/lib/node_modules/stylelint/lib/augmentConfig.js:30:12
```

**Comment by hanakin** [comment]
hmm I do not use phpstorm, but this is one of the issue I was getting with having the pkg.json file in phpBB/phpBB which is one reason why I moved it as that seemed to fix my issues with finding it. but you say it works fine with local and find in project?

**Comment by VSEphpbb** [comment]
Locally yes. Maybe it's an issue with the global installs I did in which i simply ran:
`npm install stylelint`
`npm install stylelint-order`

I was trying to avoid having it locally because of the issue where git currently does not ignore it correctly ;)

**Comment by hanakin** [comment]
for global you ran those? you need the -g flag for global install ``npm i -g stylelint stylelint-order``

**Comment by VSEphpbb** [comment]
oh yeah, the -g flag was there of course ;)

**Comment by marc1706** [comment]
On linux one even has to use `sudo` to install the global packages. Otherwise you're greeted with a nice error about missing permissions.

**Comment by hanakin** [comment]
hmm really thats rather unclean? https://github.com/sindresorhus/guides/blob/master/npm-global-without-sudo.md <- possible fix

File: development/development/phpstorm.rst [hunk 1]
```diff
@@ -72,6 +72,8 @@ The default inspection settings should work just fine. However there are a coupl
 * phpBB uses fully qualified namespaces, so you can turn off this inspection warning **PHP > Code Style > Unnecessary fully qualified name**.
 * You may enable additional JavaScript inspections. Under **JavaScript > Code quality tools**, you can enable JSCS, JSHint, JSlint and/or ESLint. phpBB comes with configuration files for JSHint and JSCS. With these inspections enabled, you can use the configuration files provided by phpBB instead of PhpStorm's default settings. To do so, point PhpStorm to the custom configuration files under **Language & Frameworks > JavaScript > Code Quality Tools**. Point JSHint to `.jshintrc` and point JSCS to `.jscsrc`.
 
+.. warning:: Stylelint is not bundled with PhpStorm. The Stylelint tool is run through NodeJS. To use Stylelint, make sure the NodeJS framework is downloaded and installed on your computer. Then you can follow these `Instructions <https://www.jetbrains.com/help/phpstorm/2016.3/using-stylelint-code-quality-tool.html>`_.
```
Comment (DavidIQ) [inline development/development/phpstorm.rst]: Is this link outdated now? Should it be this one instead?
https://www.jetbrains.com/help/phpstorm/stylelint.html

**Comment by hanakin** [comment]
new PR https://github.com/phpbb/documentation/pull/130

**Pull request closed without merging**